Sources/TransferMethod/SelectTransferMethodTypePresenter.swift

```diff
@@ -58,6 +58,10 @@ final class SelectTransferMethodTypePresenter {
         return TransferMethodRepositoryFactory.shared.transferMethodConfigurationRepository()
     }
 
+    private var userRepository: RemoteUserRepository {
```

Comment:
Instead of RemoteUserRepository it should be UserRepository

Reply:
Can you replace this computed property to be  `lazy var`?  I replaced in my PR's too. 
```
private lazy var userRepository = {
     return UserRepositoryFactory.shared.userRepository()
}()
```

Reply:
Replaced